openreferee_server/operations.py

```diff
@@ -163,3 +168,47 @@ def process_revision(event, revision, action):
             dict(text=f"This revision has been reviewed ({action}).", internal=True)
         ],
     )
+
+
+def _can_access_action(revision, action, user_is_editor):
+    if not user_is_editor:
+        return False
+    if revision["final_state"]["name"] == "accepted":
+        if any(t["code"] == "QA_APPROVED" for t in revision["tags"]):
```

Comment:
Apart from the suggestion below, we could probably also check if this tag is defined, but I'm not sure how relevant that is.
```suggestion
    if not user_is_editor:
        return False
    available_tags = get_event_tags(session, event)
    if revision["final_state"]["name"] == "accepted":
        if any(t["id"] == available_tags["QA_APPROVED"]["id"] for t in revision["tags"]):
```

Reply:
yeah i thought about it as well but decided to go for the simple solution ;)